content/graph/BellmanFord.py

```diff
@@ -0,0 +1,25 @@
+"""
+* Auhtor: Unknown
+* Source: https://en.wikipedia.org/wiki/Bellman%E2%80%93Ford_algorithm
+* Description: Computes shortest paths from $start$. Allows negative-weighted cycles
+* Time: $O(VE)$
```

Comment:
```suggestion
* Time: O(VE)
```

This might be the issue

Reply:
I think the issue is caused by 'Auhtor' instead of 'Author'